index.js

```diff
@@ -0,0 +1,96 @@
+    main.style.visibility = '';
+  };
+
+  closeButton.addEventListener('click', closeMenu);
+}
+
+// Section with featured speakers
+
+const presenters = [
+  {
+    featuredImage: 'images/steve-jobs.jpg',
+    name: 'Steve Jobs',
+    position: 'Founder of Apple inc',
+    shortBackground: 'With an idea of improving the means of communication in the world, he was ridiculed but eventually made the largest breakthrough in the mobile phone industry by creating the so called smart-phones',
+  },
+  {
+    featuredImage: 'images/bill.jpg',
+    name: 'Bill Gates',
+    position: 'Former CEO and founder of Microsoft Corporation',
+    shortBackground: 'Despite the option to return to school after major setbacks during the creation of Microsoft Corporation, he stilled pushed idea and eventually made it. His success made him the richest man for more than a century.',
+  },
+  {
+    featuredImage: 'images/jeff.jpg',
+    name: 'Jeff Bezoz',
+    position: 'Former CEO and founder of Amazon',
+    shortBackground: 'Bezoz built his company amazon from a garage together with some of his friends. Despite being laughed on his work, he kept pushing and eventually became the world richest man the year 2020.',
+  },
+  {
+    featuredImage: 'images/facebook.png',
+    name: 'Mark Zuckerburg',
+    position: 'CEO of facebook and other major social media apps',
+    shortBackground: 'Creator of the famous app facebook. After getting the idea from his friends at Havard, Mark could not let the idea die. Eventually, facebook became a success and he went to dropout of Havard to manage facebook.',
+  },
+  {
+    featuredImage: 'images/warren.jpg',
+    name: 'Warren Buffett',
+    position: 'Owner of various companies and biggest investor for cocacola',
+    shortBackground: 'With the skill of analysing business data, Warren has made major investments to businesses which grew to fortune 500 companies hence the nickname the ORACLE OF OMAHA',
+  },
+  {
+    featuredImage: 'images/adam.jpg',
+    name: 'Adam Sandler',
+    position: 'Owner and founder of golf studios',
+    shortBackground: 'With lack of funding to support his crazy acting career, Adam sought to create his own film company so as to cater for the many actors with no place to go due to lack of funding. He is now rising to riches',
+  },
+];
+
+let i = 0;
+presenters.forEach(() => {
+  i += 1;
+
+  const sectionPresenter = document.createElement('div');
+  sectionPresenter.classList.add('sec-3-presenter');
+  sectionPresenter.innerHTML = `
+  <div class="presenter-img">
+    <img src='${presenters[i - 1].featuredImage}' alt="presenter" class="presenter-1">
+  </div>
+  <div class="presenter-text">
+    <h5 class="presenter-name">${presenters[i - 1].name}</h5>
+    <p class="presenter-position">${presenters[i - 1].position}</p>
+    <p class="presenter-description">${presenters[i - 1].shortBackground}</p>
+  </div>
+  `;
+  document.querySelector('.sec-3-container').appendChild(sectionPresenter);
+});
```

Comment:
Nice job rendering the `speakers` section dynamically😉. But let's try to match the given design😉. From the design, the speaker section consist of two columns each and not three as you have used😉.

yours 
<img width="943" alt="image" src="https://github.com/2004-okari/CAPSTONE-01/assets/64643995/c5e0b519-65d9-41c6-ac81-dc16ef3ca241">

design
<img width="622" alt="image" src="https://github.com/2004-okari/CAPSTONE-01/assets/64643995/e6799c34-24f0-4ce1-93c6-d5d117cb1b0a">

also, don't forget to consider the spacings😉.